<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for codingjoe -- django-s3file</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/codingjoe/django-s3file.rss</link>
    <description>The latest CVEs for codingjoe -- django-s3file</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Sat, 03 Aug 2024 20:22:08 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2022-24840|2024-08-03T04:20:50.677Z -- codingjoe -- django-s3file
</title>
      <link>https://www.cve.org/CVERecord?id=CVE-2022-24840</link>
      <description>django-s3file is a lightweight file upload input for Django and Amazon S3 . In versions prior to 5.5.1 it was possible to traverse the entire AWS S3 bucket and in most cases to access or delete files. If the `AWS_LOCATION` setting was set, traversal was limited to that location only. The issue was discovered by the maintainer. There were no reports of the vulnerability being known to or exploited by a third party, prior to the release of the patch. The vulnerability has been fixed in version 5.5.1 and above. There is no feasible workaround. We must urge all users to immediately updated to a patched version.</description>
      <guid isPermaLink="false">CVE-2022-24840|2024-08-03T04:20:50.677Z</guid>
      <pubDate>Mon, 06 Jun 2022 19:10:11 +0000</pubDate>
    </item>
  </channel>
</rss>
